### PR TITLE
optimized-routes-backend

### PR DESCRIPTION
**Pull request by YaokunLin**

### CODE REVIEW

**Comment by YaokunLin** [comment]
ticket added: https://www.pivotaltracker.com/n/projects/2667313/stories/186043259

File: app_vue/src/utils/optimizeRouteHelper.js [hunk 1]
```diff
@@ -22,73 +24,36 @@ export const getNewOptimizedRoute = (selectedRouteList, newRouteIndexOrder) => {
   return [...intermediates];
 };
 
-const getGoogPayload = (selectedRouteList, originAddress, destinationAddress, optimize = true) => {
```
Comment (monicakochofar) [inline app_vue/src/utils/optimizeRouteHelper.js:25]: yay for less code!

**Review by monicakochofar — APPROVED** [review]
Frontend looks great! nice one

**Review by mikevespi — APPROVED** [review]
👍 Just the new line at the EoF

**Pull request merged**